.ackrc

```diff
@@ -0,0 +1 @@
+--ignore-dir=target
```

Comment:
This shouldn't be here. We all use various local tools in our software development, and these vary from developer to developer (e.g. different editors, IDEs, search tools, etc.). Files created by such local tools that are unrelated to the build should not be committed to the repository. One solution may be to add `.ackrc` to a global gitignore on your local system, as described here: https://stackoverflow.com/questions/7335420/global-git-ignore

(By this logic, the Vim-specific `*.swp` should also not be in the committed `.gitignore`. I got better about scrubbing local concerns after the initial work on this project.)